arangod/Replication/DatabaseTailingSyncer.cpp

```diff
@@ -257,7 +262,7 @@ bool DatabaseTailingSyncer::skipMarker(VPackSlice const& slice) {
   // now check for a globally unique id attribute ("cuid")
   // if its present, then we will use our local cuid -> collection name
   // translation table
-  VPackSlice const name = slice.get("cuid");
+  VPackSlice const name = slice.get(::cuidRef);
```

Comment:
wtf why is this faster ?

Reply:
This is only a very minor thing, but it allows avoiding repeated calls to ctor of velocypack::StringRef(char const*), which needs to calculate the `strlen` of the input string every time it is invoked.